README.md

```diff
@@ -112,6 +112,25 @@ These configurations should all be runnable for local testing, except for `compl
 You will need access to the vault to run the `githubauth` configuration, which requires secrets for the github oauth2 client app
 details.
 
-### Notes
+## Running locally
```

Comment:
I think you should retain the third `#` if you want "For example" to relate to the previous paragraph.

You might want to add a callback like "First, install from local sources as above" linking to 'Install from local sources'. Alternatively, put this section as a sub-section of that.

Reply:
oh I should probably just remove the section on local installation tbh

Reply:
I disagree - I don't know how to run this in development without installing from local sources, and I don't know how to install from local sources without this part of the README. I see you've kept it in though.

Reply:
OK, I've now learnt that installing from local sources is unnecessary. So I want to update the README to make the best dev workflow obvious.

Reply:
Well, actually, I think it's fine now, it was just such a sparse README before that I invented my own way of doing things, but now there is correct guidance for the dev workflow.